### PR TITLE
Chameleon Web Vest / Armored Chameleon Kit

### PR DESCRIPTION
**Pull request by FungiFellow**
## About the PR

I made a Chameleon Web Vest and priced it at 6TC. A Armored Chameleon Kit is 12TC and includes Chameleon Syndie mask, Combat Gloves, and a Helmet.

A few hypothetical scenarios are stealth ops nukies, passive syndies that want extra survivability, active syndies that want to wear funny outfits with actual protection or to be able to shed a bit of suspicion after fleeing.
## Why / Balance

A Chameleon Webvest for double the cost of a regular one minus the Valid Salad Dressing, also it be dripping. I think the Armored Chameleon kit will be a mainstay for passive syndies that want survivability
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase-

image
image
image
image
image
image

## Breaking changes

**Changelog**

:cl:
- add: Added Chameleon Webvest for 6TC
- add: Added Armored Chameleon Kit for 12TC

### CODE REVIEW

**Comment by ARMOKS** [comment]
Actually goaded item, finally we can have silly syndicate outfits that also protect.

**Comment by UbaserB** [comment]
i would buy this

**Comment by KingOnBudget** [comment]
stealth op nukies will love this.

**Comment by K-Dynamic** [comment]
That's a lot of resistance without obvious telegraphing

**Comment by ARMOKS** [comment]
> That's a lot of resistance without obvious telegraphing

Yes however its also a huge lump of your TC and very obvious once sec gets a few shots off

**Comment by K-Dynamic** [comment]
> Yes however its also a huge lump of your TC and very obvious once sec gets a few shots off

(Talking about the 6 tc cham webvest alone)

My question is how balanced it is for 6 tc, 70% pierce resist and chameleon property

Normal web vest outs you as valid but taking it off means relying on stealth without protection

Cham web vest lets you be both stealthy and protected to an absurd degree, without the inconvenience of storing a web vest and equipping delay

Other benefits include:
- Tricking people with false stats (or lackthereof) - not every secoff or command will be aware you have 70% pierce protection, especially when disguised
- Exiting firefights will be safer as you are much less likely to draw attention or crew callouts

**Comment by FungiFellow** [comment]
> Cham web vest lets you be both stealthy and protected to an absurd degree, without the inconvenience of storing a web vest and equipping delay


Yes, it's a 100% price increase over the regular version.

**Comment by FungiFellow** [comment]
> Yes, it's a 100% price increase over the regular version.

It's nowhere near absurd when Sec can just keep track of your race, any outstanding features and gender. Security can also use the power of observation to tell that they should use stuns or switch weapons instead of bullets

**Comment by Hmeister-real** [comment]
i like this 👍

**Comment by FungiFellow** [comment]
Since this hasn't been reviewed yet, I might add a Chameleon softsuit with 0 movement penalty and a vulnerability to heat.

**Comment by K-Dynamic** [comment]
Someone on discord had an interesting idea on having camo be revealed when taking damage

**Comment by deltanedas** [comment]
prop hunt ss14

**Comment by FungiFellow** [comment]
> Someone on discord had an interesting idea on having camo be revealed when taking damage

I feel that this would get Meta'd incredibly easily.

**Comment by Hmeister-real** [comment]
> prop hunt ss14

ss14 would work so well with prophunt its not even funny

**Comment by FungiFellow** [comment]
> > prop hunt ss14
> 
> ss14 would work so well with prophunt its not even funny

Mimics do exist

**Comment by FungiFellow** [comment]
That's an issue with mimics directly being named mimics

**Comment by deltanedas** [comment]
anyway chameleon breaking on damage would be metad so hard, seccies would run around punching everyone or using 1 damage spoons to chameleon check

**Comment by wrexbe** [comment]
If not "break cloak on damage", you could also consider "break cloak on attack"

With either choice, you can adjust how much damage given/taken is needed to make the cloak fail. It doesn't have to fail because of a punch

**Comment by FungiFellow** [comment]
> If not "break cloak on damage", you could also consider "break cloak on attack"
> 
> With either choice, you can adjust how much damage given/taken is needed to make the cloak fail. It doesn't have to fail because of a punch

I'm not going to add arbitrary code for it that results in it getting actively meta'd when some shitter 'accidentally' hits you with an item that meets the minimum damage making you valid or when a person misclicks.

**Comment by wrexbe** [comment]
> The penalty of the item is the increased cost over the regular version.
The cost to the attacker isn't the issue here.

> And people can logically tell if you're more tanky than you should be
Not really, by the time someone figures it out, they are probably going to be dead already.

The item needs to be fun for the attacker, and the defender. This item will give the defender the feeling they ran into a bug, or were cheated somehow. You want to prevent meta gaming, but not knowing that this item exists would be a huge disadvantage to the defender. 

From Discord
> @EmoGarbage
> Just from a game design perspective, someone tanking a bunch of extra damage for no apparent reason is very sus

**Comment by FungiFellow** [comment]
> > @EmoGarbage
> > Just from a game design perspective, someone tanking a bunch of extra damage for no apparent reason is very sus

Sus as in suspicious, should you be subjected to a search, they can discover the armor value easily

**Comment by wrexbe** [comment]
>  should you attract attention and be subjected to a search, they can discover the armor value easily.

So the solution for Sec not meta-gaming by hitting you to find the cloak, is Sec meta-gaming by looking at armor values to find the cloak. That also does not solve the problem of it not being practical to check armor values in the middle of a fight.

Let's think about this restriction though: "Camouflage breaks on the wearer attacking"
1. It's hard to meta, because you have to choose to break your own camouflage
2. If you get attacked by someone wearing a cloak, the cloak will fail, and you will have a chance to get away.
3. If you are attacked while wearing a cloak, you can decide if your going to run away, but keep your disguise, or reveal yourself.

Regardless of what I think about it though, Im not the right person to decide. I think @EmoGarbage404 is?

**Comment by superjj18** [comment]
Only issue with this is armor that people don’t know is armor.  There should be some feedback to gauge weapon effectiveness similar to how wall damage works

**Comment by ElPidor** [comment]
very good PR, do not listen to people against it. Metagaming will always be, its like whatever

**Comment by FungiFellow** [comment]
Having to update these all due to semi recent changes to upstream

**Comment by FungiFellow** [comment]
> Otherwise a regular webvest will never see action again, this is just too superior.

It has identical stats to a webvest, if I am going loud it is better to buy the classic because of its lower cost leaving me more room to buy a weapon or other goodies. I can simply buy a toolbox for insuls/flash protection. 

The perk of the Chameleon stuff is that it isn't instantly valid. But even then the web vest is near wholly covered up by webbing.

File: Resources/Prototypes/Entities/Clothing/OuterClothing/specific.yml [hunk 1]
```diff
@@ -18,3 +18,35 @@
...
+      interfaces:
+        enum.ChameleonUiKey.Key:
+          type: ChameleonBoundUserInterface
+    - type: Armor
+      modifiers:
+        coefficients:
+          Blunt: 0.6 #ballistic plates = better protection
+          Slash: 0.6
+          Piercing: 0.3
+          Heat: 0.9
```
Comment (Veritius) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/specific.yml:49]: This is identical to a web vest, making it a direct upgrade (apart from cost).
There's not really any sacrifice you have to make, especially if you're like me and barely use TC anyway.

File: Resources/Prototypes/Entities/Clothing/Masks/specific.yml [hunk 2]
```diff
@@ -24,6 +24,41 @@
...
+      - Snout
+    - type: FlashImmunity
+    - type: EyeProtection
+    - type: Armor
+      modifiers:
+        coefficients:
+          Blunt: 0.95
+          Slash: 0.95
+          Piercing: 0.95
+          Heat: 0.95
```
Comment (Veritius) [inline Resources/Prototypes/Entities/Clothing/Masks/specific.yml:60]: Does this even really do anything? I guess it saves you from crit from a few seconds if you get shot with exactly 100 pierce damage... but you'll fall to bleeding anyway.

**Review by Veritius — COMMENTED** [review]
I feel like the fact that chameleon gear gave no protection was a really interesting downside. Instead of getting shot, you could try to RP or misdirect your way out of situations. I guess survivability is good for traitors, but you can also just put on real armor if you start getting attacked.

File: Resources/Prototypes/Entities/Clothing/OuterClothing/specific.yml (hunk 1 above)
Comment (FungiFellow) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/specific.yml:49]: My reasoning was to give protection to stealth syndies, Security can still tell that something is wrong if a target seems too tanky. There is no reason for a syndicate that's going loud to use the chameleon version. To put off Sec's sens until they get a few shots off they can easily have a webbing or chest rig (Several mapped in gamer loot locations, or just take from a dead seccie)

File: Resources/Prototypes/Entities/Clothing/Masks/specific.yml (hunk 2 above)
Comment (FungiFellow) [reply Resources/Prototypes/Entities/Clothing/Masks/specific.yml:60]: Something is better than nothing, and that's not the main appeal of the mask. The main appeal lies in the Eye Protection, the fact that armor applies multiplicatively means it does barely anything.

**Comment by mirrorcult** [comment]
i dont really know if i see the point of this. isnt chameleon stuff not having armor like part of the point, what is the disadvantage here, what gameplay is created

**Comment by K-Dynamic** [comment]
A target that seems tanky is already hard to judge when most players don't memorise bullet damage or when traitors load up on meds, not to mention disguising right after a fight and losing valid status lets you roam freely (we seriously need blood contamination and splattering)

Suffice to say I'd rather purchase this than a bloodred hardsuit

**Comment by EmoGarbage404** [comment]
I think this is generally just an overly dominant concept. Being able to overpower others while being stealthy AND being able to disguise to slip away is just an incredibly frustrating playstyle being enabled very easily.

**Comment by K-Dynamic** [comment]
> Suffice to say I'd rather purchase this than a bloodred hardsuit

I should clarify that I wouldn't be opposed to *some* protection in chameleon gear, just around like 15% to 20% in total rather than a whopping 70% on pierce on vest alone

**Pull request closed without merging**